mavros_extras/src/plugins/px4flow.cpp

```diff
@@ -48,14 +51,22 @@ class PX4FlowPlugin : public MavRosPlugin {
 			ros::NodeHandle &nh,
 			diagnostic_updater::Updater &diag_updater)
 	{
-		bool mode_tx;
 		uas = &uas_;
 
-		nh.param("optical_flow_tx", mode_tx, false);
-		if (!mode_tx)
-			flow_pub = nh.advertise<mavros_extras::OpticalFlow>("optical_flow", 10);
-		else
-			flow_sub = nh.subscribe("optical_flow", 10, &PX4FlowPlugin::send_flow_cb, this);
+		flow_nh = ros::NodeHandle(nh, "px4flow");
+		
+		flow_nh.param<std::string>("frame_id", frame_id, "px4flow");
+
+		//Default rangefinder is Maxbotix HRLV-EZ4
+		flow_nh.param("ranger_type", ranger_type, 0);	// See sensor_msgs/Range docs 
+		flow_nh.param("ranger_fov", ranger_fov, 0.0); 	// TODO
+		flow_nh.param("ranger_min_range", ranger_min_range, 0.3);	

```

Comment:
In schematic says that it is MB1040: http://www.maxbotix.com/Ultrasonic_Sensors/MB1040.htm